packages/app-permissions/__tests__/index.js

```diff
@@ -0,0 +1,46 @@
+import { readFileSync } from 'fs'
+import { EMPTY_DEFINITIONS, mergeGrantsToDetails } from '../lib'
+
+const readJsonFixture = filename => {
```

Comment:
Why not simply include the test values in this file? It shouldn't need such a setup.

Reply:
I had a much larger test before, but pared it down. I think there may be other fixtures.

Reply:
Please import the objects in the test cases for now, it makes things more readdable. Doing synchronous I/O in node is not ideal, even when it's just for tests.

Reply:
Ok